src/main/java/simpaths/model/SimPathsModel.java

```diff
@@ -584,6 +584,9 @@ public void buildSchedule() {
         yearlySchedule.addCollectionEvent(persons, Person.Processes.LifeSatisfaction2);
 
 
+        // Update financial distress
+        addCollectionEventToAllYears(persons, Person.Processes.FinancialDistress);
```

Comment:
Brief note - I'd suggest putting these before the MCS/PCS/Mental health stage 2 estimates. Financial distress as a product of employment and income changes would be a causal mediator of effects on mental health?